packages/circle-react-elements/src/components/ChainIcon/ChainIcon.tsx

```diff
@@ -0,0 +1,16 @@
+import { NetworkIcon } from '@web3icons/react';
+
+import { BlockchainToIconMap } from '../../lib/constants';
```

Comment:
I'd keep BlockchainToIconMap definition in this file as it is only used here internally, no need to export it anywhere else

Reply:
fixed